workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -3,13 +3,21 @@
 using Improbable.Gdk.Core;
 using Improbable.Gdk.PlayerLifecycle;
 using Improbable.Gdk.TransformSynchronization;
+using Improbable.PlayerLifecycle;
 
 namespace Playground
 {
     public static class PlayerTemplate
     {
-        public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improbable.Vector3f position)
+        public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improbable.Vector3f position,
+            Option<byte[]> playerCreationArguments)
         {
+            if (playerCreationArguments.HasValue)
+            {
+                var test = PlayerLifecycleHelper.DeserializeParams<PlayerCreationParams>(playerCreationArguments.Value);
```

Comment:
Can this fail?

Reply:
hmm it will silently fail but return the default `PlayerCreationParams` as if nothing bad happened

I think it's worth changing serialize/deserialize to return a bool to indicate success, passing the outback back with an `out` param